### ISSUE TITLE
Default view on open

### ISSUE THREAD

**Issue by cbovis** [jacogr/atom-git-control#195]
My most common workflow when using git-control tends to be,

1. Toggle git-control
2. Hit 'compare' option
3. Review changes
4. Commit

When I open up git-control I'm presented with changed files but the right hand panel is always empty. I then have to hit the control button to see what the actual changes are within those files.

Thoughts on making the view default to the comparison page on toggle? I'm happy to put in a pull request...

**Comment by jacogr** [comment]
Good idea actually. (Honestly very short on time at, if you can make a PR, will pull it in)
